etc/kayobe/kolla/config/grafana/dashboards/ceph/ceph_mds.json

```diff
@@ -0,0 +1,316 @@
+{% raw %}
```

Comment:
As mentioned in the previous comment, config files in `etc/kayobe/kolla/config` won't get pulled into kayobe environments. The solution/workaround for this has been to symlink them, but there is a [bug](https://review.opendev.org/c/openstack/kayobe/+/803882) associated with that that may need fixing it we are going to rely on that method. We'd also need clear docs on how to do it.

Alternatively we can merge and backport Will's patches to merge these configs. Unclear if we have time to do that though.

Reply:
We can't do anything about this issue here, and it's good to have this config in place even if we need to symlink to it from an environment directory.